pdns/recursordist/test-syncres_cc8.cc

```diff
@@ -698,6 +698,7 @@ BOOST_AUTO_TEST_CASE(test_dnssec_rrsig_negcache_bogus_validity) {
   size_t queriesCount = 0;
   const time_t fixedNow = sr->getNow().tv_sec;
 
+  #if defined(HAVE_DNS_OVER_TLS) || defined(HAVE_DNS_OVER_HTTPS)
```

Comment:
This is really not the right way.

You should remove the unused vars from between the square brackets

Reply:
I changed it. Can you check if it's good now?

Reply:
Now it's updated correctly. Can you have a look at it?

Reply:
This looks good., although I do not understand why mu clang (7.0.1) does not warn about `target`.
Also: can you please squash the commits into a single one? There's nothing from your earlier versions we would like to preserve.

Reply:
i tried to squash the commits, but as you can see: it didn't work too well. Could you help me with that? I'm kinda new to forks.

Reply:
I'll just make another pull request. That's easier